### PR TITLE
win_feature improvements

### PR DESCRIPTION
**Pull request by trondhindenes**
Various improvements to win_feature. #8068 plus some enhancements when specifyign a comma-separated list of windows features to process.

### CODE REVIEW

File: windows/win_feature.py [hunk 1]
```diff
@@ -87,11 +87,24 @@
   tasks:
     - name: Install IIS
       win_feature:
-        name: "Web-Server"
-        state: absent
+        name: "Web-Server,Web-Mgmt-Tools,web-Default-Doc"
+        state: Present
```
Comment (cchurch) [inline windows/win_feature.py]: Should be lowercase to be consistent with what the module is expecting in https://github.com/trondhindenes/ansible-modules-core/blob/fca52e6f16a61756b7183aaa748aa9567e448410/windows/win_feature.ps1#L84

File: windows/win_feature.py [hunk 2]
```diff
@@ -87,11 +87,24 @@
...
         include_management_tools: yes
 
+- name: Install IIS
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Install IIS
+      win_feature:
+        name: "Web-Server","Web-Mgmt-Tools","web-Default-Doc"
+        state: Present
```
Comment (cchurch) [inline windows/win_feature.py]: Lowercase, same as above.

**Comment by cchurch** [comment]
Ignore the comments made in the code; I'd glanced over the part where it converts $state to lowercase in https://github.com/trondhindenes/ansible-modules-core/blob/fca52e6f16a61756b7183aaa748aa9567e448410/windows/win_feature.ps1#L37

It does need a rebase, however.

**Comment by gregdek** [comment]
Hey @trondhindenes -- want to make sure this one is still valid / on your radar / worth rebasing.

**Comment by trondhindenes** [comment]
Hi,
I don't have that branch locally anymore, so I'll close this and open a new one.

**Pull request closed without merging**